trade-finance-manager-api/src/graphql/resolvers/filters/filterActivities.js

```diff
@@ -0,0 +1,27 @@
+const FILTER_TYPE = {
+  ALL: 'all-activity',
+  COMMENT: 'comments-only',
+};
+
+const filter = (activities, FILTER_VALUE) => activities.filter((activity) => activity.type === FILTER_VALUE);
+
+const filterByComment = (activities) => filter(activities, 'COMMENT');
+
+
+const filterActivities = (activities, filtersObj) => {
+  if (!filtersObj || !filtersObj.filterType || filtersObj.filterType === FILTER_TYPE.ALL) {
+    return activities;
+  }
+
+  const { filterType } = filtersObj;
+
+  if (filterType === FILTER_TYPE.COMMENT) {
+    return filterByComment(activities);
+  }
+
+  return activities;
+};
+
+module.exports = {
+  filterActivities,
+};
```

Comment:
need tests for these functions, there is example with `filterTasks` is same directory

Reply:
done